### ISSUE TITLE
Redshift driver configured with AutoCreate fails without explanation

### ISSUE THREAD

**Issue by jorhett** [aws/aws-toolkit-jetbrains#3062]
**Describe the bug**

The Jetbrains Redshift driver includes a setting for `AutoCreate`.  Setting this value does not change the API call made because it's hardcoded to off at https://github.com/aws/aws-toolkit-jetbrains/blob/main/jetbrains-ultimate/src/software/aws/toolkits/jetbrains/services/redshift/auth/IamAuth.kt#L85-L86

**To reproduce**

1. Configure a Database connection with an IAM profile that has CreateDatabaseUser credentials
2. Go to Advanced Settings and configure AutoCreate to True, as documented here https://docs.aws.amazon.com/redshift/latest/mgmt/jdbc20-configuration-options.html#jdbc20-autocreate-option

Try to connect, receive: `[42704] FATAL: user "jorhett" does not exist`

3. Use a different product that calls the API correctly, including `aws redshift` CLI
4. Confirm that Datagrap now connections with no difficulty

**Expected behavior**

If the AutoCreate parameter is enabled it should be passed through to the driver.

**Screenshots**

As shown by CloudTrail when using Datagrip:

```
  "requestParameters": {
        "dbUser": "jorhett",
        "clusterIdentifier": "test-redshift",
        "autoCreate": false
    },
```

**Your Environment**

- OS: macOS
- JetBrains product: Datagrip
- JetBrains product version: 2021.3.4
- AWS Toolkit version: 1.37-213

**Additional context**

It doesn't appear you pass DbGroups either, even though that is also a parameter available from the driver.

**Comment by kiiadi** [comment]
Did you want to have a go at submitting a pull request for this? Seems like a pretty straightforward fix.

**Comment by jorhett** [comment]
> Did you want to have a go at submitting a pull request for this? Seems like a pretty straightforward fix.

No. I don't work in this language and it seems like something Amazon should be fixing for paying customers
